sunkit_spex/legacy/thermal.py

```diff
@@ -154,7 +154,7 @@ def setup_line_parameters(filename=None):
         "It (and its unit) therefore must be multiplied by emission measure and "
         "divided by 4 * pi * observer_distance**2 to get observed values."
     )
-    line_grid["line peaks keV"] = (line_info.peak_energy.data * line_info.attrs["units"]["peak_energy"]).to_value(
+    line_grid["line peaks keV"] = (line_info.peak_energy.data << line_info.attrs["units"]["peak_energy"]).to_value(
```

Comment:
So my guess is that probably don't even need to add the unit as the only was to get   keV$^2$ is if `line_info.peak_energy.data` is already in `keV`

Reply:
Right, but the prior version of astropy (?) didn't put units by default. Kind of strange